tests/Resources/Common.robot

```diff
@@ -106,6 +106,7 @@ URLs HTTP Status Code Should Be Equal To
     [Arguments]    ${link_elements}    ${expected_status}=200    ${timeout}=20
     FOR    ${idx}    ${ext_link}    IN ENUMERATE    @{link_elements}    start=1
         ${href}=    Get Element Attribute    ${ext_link}    href
+        ${text}=    Get Text    ${ext_link}
```

Comment:
you don't use this anywhere

Reply:
I know, I've added it for debug reasons, to compare what the code is capturing with what's in the UI